examples/mobile/UIComponents/components/etc/progress/progress-circle/progresscircle.html

```diff
@@ -3,9 +3,9 @@
 
 <head>
 	<meta content="width=device-width, user-scalable=no" name="viewport" />
-	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../../css/style.css" rel="stylesheet" />
-	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	<link href="../../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../../lib/tau/mobile/js/tau.js">
 	</script>
```

Comment:
in line 30, add class "ui-bottom-button" to footer

Reply:
Done